### ISSUE TITLE
SSL: CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by gentunian** [dj-wasabi/ansible-zabbix-agent#66]
I have a valid certificate provided by godaddy but I got this error:

```
fatal: [10.168.3.101 -> localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to connect to Zabbix server: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)>"}
```

Is there an option to set `no_verify` option?

**Comment by d47zm3** [comment]
Hey, I had same issue, please try to export this environment variable before running playbook
export PYTHONHTTPSVERIFY=0
it worked for me

**Comment by gentunian** [comment]
Thanks! that worked.

**Comment by dj-wasabi** [comment]
I created a PR for this a while back: https://github.com/ansible/ansible/pull/24595
Not yet merged.

**Comment by dj-wasabi** [comment]
PR is merged and is available with Ansible 2.5. (I have no idea when this is released though.)
